test/index.js

```diff
@@ -668,8 +668,9 @@ test ('equals', function() {
   eq (Z.equals (Math.sin, Math.cos), false);
   eq (Z.equals (Identity (Identity (Identity (0))), Identity (Identity (Identity (0)))), true);
   eq (Z.equals (Identity (Identity (Identity (0))), Identity (Identity (Identity (1)))), false);
-  eq (Z.equals ({'@@type': 'my-package/Quux@1'}, {'@@type': 'my-package/Quux@1'}), true);
+  eq (Z.equals ({'@@type': 'my-package/Quux@1'}, {'@@type': 'my-package/Quux@1'}), false);
   eq (Z.equals (Array.prototype, Array.prototype), true);
+  delete Maybe['@@type'];
```

Comment:
This `delete` statement is required to prevent `Maybe` from being treated as a member of a type that lacks a `fantasy-land/equals` method. I will remove it when I upgrade the sanctuary-maybe dependency.

Reply:
#135